### PR TITLE
test(gui): fix flaky spaces test suite

### PR DESCRIPTION
**Pull request by saw-jan**
- Catch the webdriver error and treat it as element not found. There can be an error even if the element is visible due to reload or refresh.
- split gui test suites

### CODE REVIEW

**Comment by saw-jan** [comment]
^ @Asmitapaudel

File: .woodpecker/ui-tests.yaml [hunk 1]
```diff
@@ -38,7 +38,9 @@ matrix:
     - MATRIX_NAME: gui-tests-1
       SUITES: 'features/activity features/add-account features/delete-files-folders features/edit-files features/login-logout features/move-files-folders'
     - MATRIX_NAME: gui-tests-2
-      SUITES: 'features/remove-account-connection features/spaces features/sync-resources features/tabs-settings features/vfs'
+      SUITES: 'features/remove-account-connection features/spaces features/sync-edge-cases features/tabs-settings'
+    - MATRIX_NAME: gui-tests-3
+      SUITES: 'features/sync-resources features/vfs'
```
Comment (prashant-gurung899) [inline .woodpecker/ui-tests.yaml:43]: We could also add few more test suites here and distribute among pipelines equally.

File: test/gui/features/sync-edge-cases/syncResources.feature
Comment (prashant-gurung899) [inline test/gui/features/sync-edge-cases/syncResources.feature:1]: could we change the filename for this one?
Using same name will be confusing.

Comment (saw-jan) [reply test/gui/features/sync-edge-cases/syncResources.feature:1]: IMO, it should be fine. suite name will differentiate the type. The test scenarios are actually for syncing resources

File: .woodpecker/ui-tests.yaml (hunk 1 above)
Comment (saw-jan) [reply .woodpecker/ui-tests.yaml:43]: all runs for about ~10 minutes. looks pretty balanced for now. sync-resources suite has more scenarios

Comment (saw-jan) [reply .woodpecker/ui-tests.yaml:43]: but may be we can add small suite form group 1

Comment (prashant-gurung899) [reply .woodpecker/ui-tests.yaml:43]: Yeah, the run time for all groups seem to be around 10 minutes. So, we can keep it as it is for now.

**Comment by prashant-gurung899** [comment]
LGTM 👍🏼

**Review by prashant-gurung899 — APPROVED** [review]
